src/fragments/lib/auth/js/react-native-social.mdx

```diff
@@ -187,27 +187,39 @@ For *Sign in Redirect URI(s)* inputs, you can put one URI for local development
 **Note:** if you have multiple redirect URI inputs, you'll need to handle both of them where you configure your Amplify project. For example:
 
 ```javascript
-import awsConfig from "./aws-exports";
+import amplifyconfig from "./amplifyconfiguration.json";
 
 const isLocalhost = Boolean(__DEV__);
 
 // Assuming you have two redirect URIs, and the first is for localhost and second is for production
 const [
   localRedirectSignIn,
   productionRedirectSignIn,
-] = awsConfig.oauth.redirectSignIn.split(",");
+] = amplifyconfig.Auth.Cognito?.loginWith?.oauth?.redirectSignIn?.split(",");
 
 const [
   localRedirectSignOut,
   productionRedirectSignOut,
-] = awsConfig.oauth.redirectSignOut.split(",");
+] = amplifyconfig.Auth.Cognito?.loginWith?.oauth?.redirectSignOut?.split(",");
```

Comment:
Are these keys an available in the autogenerated config? I thought they remain the old "snake_case" format.